woocommerce/woocommerce-gateway-stripe
# [UPE - old settings] Move express checkout size option to PRB section

# Changes proposed in this Pull Request:

Resolves #1897 

This PR basically moves the option **Size of the button displayed for Express Checkouts**, which is currently on the **Checkout experience** section, to the **Payment Request Buttons** section.

With the move, the option visibility is no longer conditioned to the UPE flag check.

# Testing instructions
Before testing make sure you have the `_wcstripe_feature_upe_settings` option set to `no` in the database.

1. Visit http://localhost:8082/ and log in as admin.
2. Go to `WooCommerce` > `Settings` > `Payments` > `Stripe UPE`.
3. **Size of the button displayed for Express Checkouts** option should be below **Payment Request Button Locations** option as well as it must be working.

![image](https://user-images.githubusercontent.com/10233985/134013646-e61d7fb7-6d5d-40c5-b2dc-27c1752c8dc0.png)


-------------------
